src/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
Could you justify the need for this change?

Reply:
https://dequeuniversity.com/rules/axe/2.2/html-lang?application=lighthouse

Reply:
I think this reference justifies your change :) @subhahu123

Reply:
@subhahu123 Cool :+1: Good catch.